source/documentation/getting_started/get_started.erb

```diff
@@ -16,6 +16,24 @@ Once you have a personal account, you can access the GOV.UK PaaS admin tool for
 
 Contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) if you have any questions.
 
+### Trial and paid accounts
+
+When your department, agency or team first [signs up to the GOV.UK PaaS](https://www.cloud.service.gov.uk/signup), their account is a free trial account.
+
+With a trial account, your department, agency or team can:
+
+- manage [users](/orgs_spaces_users.html#users-and-user-roles)
```

Comment:
Yes, makes more sense to put this detail here than https://www.cloud.service.gov.uk/get-started

One this is published, I'd propose taking that content off the product page.

Reply:
agree